packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -269,8 +272,11 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
 
   useEffect(() => {
     toggleAllRowsSelected(false);
+    const validChars = /^[0-9.]+$/;
```

Comment:
the `.` catches every character. I would go for this regex:
```suggestion
    const validChars = /^(\d\.)*\d$/;
```